src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -1,3 +1,23 @@
-export const CommentInfo: React.FC = () => (
-  <>Put the comment here</>
+import { Comment } from '../../types/Comment';
+
+type Props = {
+  comment: Comment;
+};
+
+export const CommentInfo: React.FC<Props> = ({ comment }) => (
+  <>
+    <div className="CommentInfo__title">
+      <h3 className="CommentInfo__name">{comment.name}</h3>
+
+      {' by '}
```

Comment:
```suggestion
      by
```
you can just write the plain text 😉